README.md

```diff
@@ -1,654 +1,43 @@
 # MarkLogic Kubernetes Helm Chart
 
-- [MarkLogic Kubernetes Helm Chart](#marklogic-kubernetes-helm-chart)
-- [Introduction](#introduction)
-- [Prerequisites](#prerequisites)
-  - [Set Up the Required Tools](#set-up-the-required-tools)
-    - [Helm](#helm)
-    - [Kubectl](#kubectl)
-  - [Set Up the Kubernetes Cluster](#set-up-the-kubernetes-cluster)
-    - [Local Development MiniKube](#local-development-minikube)
-    - [Production Workload: AWS EKS](#production-workload-aws-eks)
-      - [Install eksctl](#install-eksctl)
-      - [Using eksctl to Provision Kubernetes Cluster on EKS](#using-eksctl-to-provision-kubernetes-cluster-on-eks)
-      - [Suggestions for Naming](#suggestions-for-naming)
-- [Install MarkLogic Helm Chart](#install-marklogic-helm-chart)
-  - [Add MarkLogic Repo](#add-marklogic-repo)
-  - [Installing the Chart](#installing-the-chart)
-  - [Configuration Options](#configuration-options)
-    - [--values](#--values)
-    - [--set](#--set)
-    - [Setting MarkLogic admin password](#setting-marklogic-admin-password)
-    - [Log Collection](#log-collection)
-  - [Adding and Removing Hosts from Clusters](#adding-and-removing-hosts-from-clusters)
-    - [Adding Hosts](#adding-hosts)
-    - [Removing Hosts](#removing-hosts)
-    - [Enabling SSL over XDQP](#enabling-ssl-over-xdqp)
-- [Deploying a MarkLogic Cluster with Multiple Groups](#deploying-a-marklogic-cluster-with-multiple-groups)
-- [Access the MarkLogic Server](#access-the-marklogic-server)
-  - [Service](#service)
-    - [Get the ClusterIP Service Name](#get-the-clusterip-service-name)
-    - [Using the Service DNS Record to Access MarkLogic](#using-the-service-dns-record-to-access-marklogic)
-    - [Additional Ports](#additional-ports)
-  - [Port Forward](#port-forward)
-    - [Forward to Pod](#forward-to-pod)
-    - [Forward to Service](#forward-to-service)
-  - [HAPorxy Load Balancer](#haporxy-load-balancer)
-    - [Enable HAProxy](#enable-haproxy)
-    - [Configuration](#configuration)
-      - [ConfigMap](#configmap)
-      - [Modify Port Configuration](#modify-port-configuration)
-      - [Automatic Reload Configuration](#automatic-reload-configuration)
-      - [External Access](#external-access)
-    - [Access HAProxy](#access-haproxy)
-- [Security](#security)
-  - [Security Context](#security-context)
-  - [Network Policy](#network-policy)
-  - [Pod Priorty](#pod-priorty)
-  - [Notice](#notice)
-- [Upgrades](#upgrades)
-- [Uninstalling the Chart](#uninstalling-the-chart)
-- [Parameters](#parameters)
-- [Known Issues and Limitations](#known-issues-and-limitations)
+This repository enables you to install MarkLogic on a Kubernetes cluster using the Helm package manager. 
```

Comment:
This repository contains a Helm Chart that allows you to deploy MarkLogic on a Kubernetes cluster. Below is a brief description of how to easily create a MarkLogic StatefulSet for development and testing. See [insert link to our docs here] for detailed documentation about running MarkLogic on Kubernetes.

Reply:
updated intro